src/ORM/Locator/TableLocator.php

```diff
@@ -299,4 +306,29 @@ public function remove($alias)
             $this->_fallbacked[$alias]
         );
     }
+
+    /**
+     * Returns a singleton instance of LocatorInterface implementation.
+     *
+     * @return \Cake\ORM\Locator\LocatorInterface
+     */
+    public static function getInstance()
```

Comment:
Note that I haven't added any new trait. Just reused the existing one.

`ModelAwareTrait` is used to work with *models* in general while there were two ways to work with ORM while developing plugins for example. Using static classes shouldn't be the preferred one.